lib/helpers.js

```diff
@@ -111,10 +111,11 @@ function getJavaHome () {
 }
 
 /*
- * Checks mShowingLockscreen in dumpsys output to determine if lock screen is showing
+ * Checks mShowingLockscreen or mDreamingLockscreen in dumpsys output to determine
```

Comment:
Could you please update the docstring to include _param_ and _return_  descriptions?